libs/sdk-ui-ext/src/index.ts

```diff
@@ -76,3 +78,10 @@ export function getInsightWithAppliedDrillDown(
 export function getInsightVisualizationMeta(insight: IInsightDefinition): IVisualizationMeta {
     return FullVisualizationCatalog.forInsight(insight).getMeta();
 }
+
+/**
+ * @internal
```

Comment:
This function should be used in AD discussed with Dan (His idea and was used getInsightVisualizationMeta) to hide internals FullVisualizationCatalog calls

Reply:
Matyáš has a good point that this is only for AD usage and AD already imports the whole `FullVisualizationCatalog` anyway (or if not it can use the `/esm/internal/` import as Matyáš is suggesting). Sorry for misleading you with this approach, it is only needed if the thing is to be used by other SDK package. Please move this function to some `/internal/` place.

Reply:
Method deleted